invoices/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+class Invoice(models.Model):
+    """ Model for invoices
+    """
+    TERMS = (
```

Comment:
Make it dynamic so it can be reusable. also put simplier value on the database. `due_receipt` instead of `Due of Receipt`

```
due_receipt
due_end_next_month
due_end_month
```